internal/ips/request.go

```diff
@@ -49,22 +50,30 @@ func (c *Client) Request() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			req := &packngo.IPReservationRequest{
-				Type:     packngo.IPReservationType(ttype),
-				Quantity: quantity,
-				Facility: &facility,
+
+			req := &metal.IPReservationRequestInput{
 				Metro:    &metro,
 				Tags:     tags,
+				Quantity: int32(quantity),
+				Type:     ttype,
+				Facility: &facility,
+			}
+
+			requestIPReservationRequest := &metal.RequestIPReservationRequest{
+				IPReservationRequestInput: req,
 			}
 
-			reservation, _, err := c.ProjectService.Request(projectID, req)
+			reservation, _, err := c.IPService.RequestIPReservation(context.Background(), projectID).RequestIPReservationRequest(*requestIPReservationRequest).Execute()
```

Comment:
The following command throws an Error, but but request is accepted and allocated IP's

```
Vasubabus-MacBook-Pro:bin vasubabu$ ./metal ip request -p a48e16b7-77bd-4172-b25f-3e8633e1e171 -t public_ipv4 -m da -q 4
Error: Could not request IP addresses: data matches more than one schema in oneOf(RequestIPReservation201Response)
```